client/db/test/types_test.go

```diff
@@ -11,8 +11,9 @@ import (
 
 func TestAccountInfo(t *testing.T) {
 	spins := 10000
-	ais := make([]*db.AccountInfo, 0, spins)
-	nTimes(spins, func(int) { ais = append(ais, RandomAccountInfo()) })
```

Comment:
The `ais` slice was not used in this test as it test the round trip encode/decode, generating a random account in the same loop.